pkg/errors/config.go

```diff
@@ -0,0 +1,22 @@
+package errors
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Config returns an error specifically tailored for reporting errors with configuration
+// values. You can check for these errors by calling errors.Is(err, KindConfigError)
+// (from the github.com/gomods/athens/pkg/errors package).
+//
+// Generally these kinds of errors should make Athens crash because it was configured
+// improperly
+func Config(op Op, field, helpText, url string) error {
+	slc := []string{
+		fmt.Sprintf("There was a configuration error with %s. %s", field, helpText),
+	}
+	if url != "" {
+		slc = append(slc, fmt.Sprintf("Please see %s for more information.", url))
+	}
+	return E(op, KindConfigError, strings.Join(slc, " "))
```

Comment:
I'm actually not sure, but would it be more readable if we strings.Join'd by "\n\t"? I haven't looked at what all of those errors will look like on the terminal yet, but will git-checkout your branch to check it out 👍

Reply:
They'll be a long-ish single line most of the time. For example: `There was a configuration error with somefield. You have to do something else. Please see docs.gomods.io for more information`.

I agree that multiple lines would be more readable. I'll make that change.